lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -1419,7 +1419,7 @@ check_terminator({End, {EndLine, EndColumn, _}}, [{Start, {StartLine, StartColum
     ExpectedEnd ->
       Suffix =
         io_lib:format(
-          "\n\n    HINT: the \"~ts\" on line ~B is missing terminator \"~ts\"",
+          "\nHINT: the \"~ts\" on line ~B is missing terminator \"~ts\"",
```

Comment:
We no longer need to show this hint in my opinion and we can get rid of the whole suffix.

Reply:
Done, I personally think the hint is indeed a bit redudant, but at the same time we don't show what we were expecting, so maybe for a complete beginner it may not be obvious that a `fn` expects an `end`.

Perhaps in the future we could the missing token from the lexer and show something like `└  mismatched unclosed delimiter (expecting "end")`

Reply:
But I also like your suggestion of bringing that from the lexer and including on the mismatched one. Preferences?

Reply:
I think it looks better on the line that points to the mismatched closing delimiter. The procedure would be to put this info in the location, kinda using it as a Meta and then just pass it to the exception as a field?

Reply:
Yeah, in this case, I think we can add to open_delimiter and close_delimiter, and have both available?